phpstan.neon

```diff
@@ -22,5 +22,3 @@ parameters:
 
         # That tag is empty on purpose
         - '#PHPDoc tag @var has invalid value \(\)\: Unexpected token "\*/", expected type at offset 9#'
-        # Backwards-compatibility
-        - '#^Return type.*of method.*DocLexer::peek.*should be compatible.*$#'
```

Comment:
Phpstan warned me :see_no_evil:

Reply:
Actually, the parent type returns `array`, so it should not be happy with this :thinking:

Reply:
Ah Phpstan is only at level 3 :thinking: